src/dali/abstract_ccxt_input_plugin.py

```diff
@@ -178,69 +180,73 @@ def _process_deposits(
-                    since=pagination_details.since,
-                    limit=pagination_details.limit,
-                    params=pagination_details.params,
-                )
-                # CCXT returns a standardized response from fetch_deposits. 'info' is the exchange-specific information
-                # in this case from Binance.com
-
-                # {
-                #   'info': {
-                #       'amount': '0.00999800',
-                #       'coin': 'PAXG',
-                #       'network': 'ETH',
-                #       'status': '1',
-                #       'address': '0x788cabe9236ce061e5a892e1a59395a81fc8d62c',
-                #       'addressTag': '',
-                #       'txId': '0xaad4654a3234aa6118af9b4b335f5ae81c360b2394721c019b5d1e75328b09f3',
-                #       'insertTime': '1599621997000',
-                #       'transferType': '0',
-                #       'confirmTimes': '12/12',
-                #       'unlockConfirm': '12/12',
-                #       'walletType': '0'
-                #   },
-                #   'id': None,
-                #   'txid': '0xaad4654a3234aa6118af9b4b335f5ae81c360b2394721c019b5d1e75328b09f3',
-                #   'timestamp': 1599621997000,
-                #   'datetime': '2020-09-09T03:26:37.000Z',
-                #   'network': 'ETH',
-                #   'address': '0x788cabe9236ce061e5a892e1a59395a81fc8d62c',
-                #   'addressTo': '0x788cabe9236ce061e5a892e1a59395a81fc8d62c',
-                #   'addressFrom': None,
-                #   'tag': None,
-                #   'tagTo': None,
-                #   'tagFrom': None,
-                #   'type': 'deposit',
-                #   'amount': 0.00999800,
-                #   'currency': 'PAXG',
-                #   'status': 'ok',
-                #   'updated': None,
-                #   'internal': False,
-                #   'fee': None
-                # }
-
-                pagination_detail_iterator.update_fetched_elements(deposits)
-
-                with ThreadPool(self._thread_count) as pool:
-                    processing_result_list: List[Optional[ProcessOperationResult]] = pool.map(self._process_transfer, deposits)
-
-                for processing_result in processing_result_list:
-                    if processing_result is None:
-                        continue
-                    if processing_result.intra_transactions:
-                        intra_transactions.extend(processing_result.intra_transactions)
-
-        except StopIteration:
-            # End of pagination details
-            pass
+        pagination_detail_set: Optional[AbstractPaginationDetailSet] = self._get_process_deposits_pagination_detail_set()
+        if not pagination_detail_set:
+            self._logger.error("No Pagination Details for Deposits")
+        else:
+            pagination_detail_iterator: AbstractPaginationDetailsIterator = iter(pagination_detail_set)
```

Comment:
Remove `else` to avoid unnecessary extra level of indentation:
```suggestion
        if not pagination_detail_set:
            self._logger.error("No Pagination Details for Deposits")
            return
        pagination_detail_iterator: AbstractPaginationDetailsIterator = iter(pagination_detail_set)
        ...
```